### PR TITLE
Data science landing page

### PR DESCRIPTION
**Pull request by davidfischer**
- Unlike the "Python" landing page, this is advertiser focused, not publisher focused.
- Update the advertiser inbound form to have pre-checked topics. Data science will be pre-checked here.
- I didn't start a datascience focused house ad campaign but that might be a good idea.
- If accepted, I'll update the "hiring" landing page to use the same directory structure.

### CODE REVIEW

File: content/pages/topics/data-science.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title: Advertising for data scientists and machine learning experts
+slug: data-science
+url: advertisers/topics/data-science/
+save_as: advertisers/topics/data-science/index.html
+description: EthicalAds is the best solution to advertise to data science and machine learning professionals while respecting their privacy.
```
Comment (ericholscher) [inline content/pages/topics/data-science.md:5]: 💯  solution -- I wonder if we want `marketing` in here at all?

File: ethicalads-theme/templates/ea/topics/data-science.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  <!-- Content -->
+  <div class="container">
+    <div class="row align-items-center text-center text-md-left">
+      <div class="col-12 col-md-6">
+
+        <!-- Image -->
+        <img src="/theme/img/datascience.jpg" alt="Data Science" class="img-fluid mw-100 float-right mb-6 mb-md-0" data-aos="fade-right">
+        <p class="small text-muted">
+          <em>
+          Photo by <a href="https://unsplash.com/@ikukevk?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Kevin Ku</a> on <a href="https://unsplash.com/photos/w7ZyuGYNpRQ?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
```
Comment (ericholscher) [inline ethicalads-theme/templates/ea/topics/data-science.html:24]: Great lead photo

**Review by ericholscher — APPROVED** [review]
This looks great. I wonder if we should mention pricing anywhere, that seems like the primary question I'd have after filling out the form. Or at least a direct link to the pricing section?

**Comment by davidfischer** [comment]
We should mention pricing. I think I'll replace "What does it mean to target data scientists?" with pricing.

**Review by ericholscher — APPROVED** [review]

**Pull request merged**